native/src/init/mount.cpp

```diff
@@ -29,21 +29,23 @@ bool avd_hack = false;
 static void parse_device(devinfo *dev, const char *uevent) {
     dev->partname[0] = '\0';
     dev->devpath[0] = '\0';
+    dev->dmname[0] = '\0';
+    dev->devname[0] = '\0';
     parse_prop_file(uevent, [=](string_view key, string_view value) -> bool {
         if (key == "MAJOR")
             dev->major = parse_int(value.data());
         else if (key == "MINOR")
             dev->minor = parse_int(value.data());
         else if (key == "DEVNAME")
-            strcpy(dev->devname, value.data());
+            strscpy(dev->devname, value.data(), sizeof(dev->devname));
         else if (key == "PARTNAME")
-            strcpy(dev->partname, value.data());
+            strscpy(dev->partname, value.data(), sizeof(dev->devname));
```

Comment:
Is it intentional that this is using `sizeof(dev->devname)` instead of `sizeof(dev->partname)`? Might be irrelevant, I'm not exactly sure how this code is called, I just happened to glance at it the other day.

Reply:
It doesn't really matter, partname and devname are both 32.
@yujincheng08 Can you answer whether this is intentional or unquestionable?

I'm also worried that strscpy won't add zero terminators, maybe we should use `strncat(dst, src, buffer_size - 1)`.